src/EA.Weee.RequestHandlers.Tests.Unit/Scheme/MemberRegistration/TotalChargeCalculatorTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace EA.Weee.RequestHandlers.Tests.Unit.Scheme.MemberRegistration
+{
+    using System;
+    using Domain.Error;
+    using Domain.Scheme;
+    using EA.Weee.RequestHandlers.DataReturns.ProcessDataReturnXmlFile;
+    using EA.Weee.RequestHandlers.Scheme.Interfaces;
+    using EA.Weee.Requests.Scheme.MemberRegistration;
+    using FakeItEasy;
+    using RequestHandlers.Scheme.MemberRegistration;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Reflection;
+    using System.Text;
+    using Xml.Converter;
+    using Xunit;
```

Comment:
Can we add some unit tests for checking of when annual charge has been applied... and when not applied

Reply:
ok